components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Protocol/src/Services/SubscriptionServices.cs

```diff
@@ -76,7 +76,7 @@ internal sealed class SubscriptionWrapper : ISubscription {
                 get {
                     return (IsConnectionOk && _currentlyMonitored != null) ?
                         _currentlyMonitored
-                            .Where(x => x.Template.MonitoringMode != Publisher.Models.MonitoringMode.Disabled)
+                            .Where(x => x.Item.Status.MonitoringMode != Opc.Ua.MonitoringMode.Disabled)
```

Comment:
I think this is ok, but I would suggest testing it against the desired mode, i.e., if the desired mode is Disabled, it is actually ok.  If the desired mode != actual, then we are not monitoring.